internal/integration/rest/utils.go

```diff
@@ -1,38 +1,83 @@
 package rest
 
-import "time"
+import (
+	"fmt"
+	"regexp"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/rs/zerolog/log"
+)
 
 type Retry struct {
 	Attempts int
 	MinSleep,
 	MaxSleep time.Duration
 }
 
-// ExtendProjects adds custom fields and created date to projects
-func ExtendProjects(projects []*Project, projectsOData []*ProjectOData) []*Project {
-	for indexProject, project := range projects {
-		// arrays have to have the same order
-		if len(projectsOData) > indexProject && project.ID == projectsOData[indexProject].ID {
-			addDateAndCustomFields(project, projectsOData[indexProject])
-			continue
-		}
-
-		// if we have wrong order in OData array
-		for _, projectOData := range projectsOData {
-			if project.ID == projectOData.ID {
-				addDateAndCustomFields(project, projectOData)
-				break
-			}
-		}
-	}
-
-	return projects
+// GetFilterForProjects get filter string for projects list
+func GetFilterForProjects(fromDate, teamName, projectIds string) string {
```

Comment:
Please add unit tests.
Looks like we need at least:
 - empty team + empty projectids
 - team + empty projectids
 - empty team + project ids
 - team + project ids

Reply:
I covered such test cases